NeoContract/CNEO.cs

```diff
@@ -10,6 +10,8 @@ namespace CNEO
 {
     public class CNEO : SmartContract
     {
+        private static readonly byte[] GasClaim = "AQzRMe3zyGS8W177xLJfewRRQZY2kddMun".ToScriptHash();
```

Comment:
Testnet address?

Reply:
yes, a template for someone can control the gas generated by NEO in CNEO contract.

Reply:
Remember to change it before deploy on mainnet